native/ios/WatermelonDB/SupportingFiles/Bridging.h

```diff
@@ -1,2 +1,2 @@
-#import "../FMDB/src/fmdb/FMDB.h"
+#import "FMDB.h"
```

Comment:
that will break watermelon when not using cocoa pods

Reply:
@radex This compiled with Xcode 11.2.
I don’t know how to keep the folder structure in Cocoa pods.
See CocoaPods/CocoaPods#8873

Reply:
@sidferreira sure, give it a try